doc/stdlib/base.rst

```diff
@@ -78,21 +78,23 @@ Getting Around
 
    Return a string with the contents of the operating system clipboard ("paste").
 
-.. function:: require(file::AbstractString...)
+.. function:: require(module::Symbol)
 
-   Load source files once, in the context of the ``Main`` module, on every active node, searching standard locations for files. ``require`` is considered a top-level operation, so it sets the current ``include`` path but does not use it to search for files (see help for ``include``). This function is typically used to load library code, and is implicitly called by ``using`` to load packages.
+   This function is part of the implementation of ``using`` / ``import``, if a module is not already defined in ``Main``. It can also be called directly to force reloading a module, regardless of whether it has been loaded before (for exmple, when interactively developing libraries).
+
+   Loads a source files, in the context of the ``Main`` module, on every active node, searching standard locations for files. ``require`` is considered a top-level operation, so it sets the current ``include`` path but does not use it to search for files (see help for ``include``). This function is typically used to load library code, and is implicitly called by ``using`` to load packages.
 
    When searching for files, ``require`` first looks in the current working directory, then looks for package code under ``Pkg.dir()``, then tries paths in the global array ``LOAD_PATH``.
 
-.. function:: reload(file::AbstractString)
+.. function:: compile(module::String)
```

Comment:
There is no `compile` method that takes a string.   We have `compile(mod::Symbol)`.